app/models/event.rb

```diff
@@ -48,6 +64,13 @@ def notify_webhooks!
 
   private
 
+  def channel_object
+    if    GROUP_KINDS.include?      self.kind then self.eventable.group
```

Comment:
I think the MessageChannelService was a little bit too knowledgeable about what objects were getting passed in. This means the MCS has no idea what an event is, and only takes a model and some data to publish to that model's channel.

This switch statement is an artifact of the slightly wonky event system, but should work for now.